packages/editor-ui/src/components/CredentialCard.vue

```diff
@@ -45,6 +46,7 @@ const uiStore = useUIStore();
 const credentialsStore = useCredentialsStore();
 const projectsStore = useProjectsStore();
 
+const resourceTypeText = computed(() => locale.baseText('generic.credential').toLowerCase());
```

Comment:
nit: `resourceTypeLabel` would've been more consistent as a name in all these places.

Reply:
Yes, makes sense, done